init-tools.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+if [ "$BUILDVARS_DONE" != 1 ]; then
+    . ./buildscripts/buildvars-setup.sh
```

Comment:
buildvars-setup does more than just setup env variables. It also creates directories. It would be nice to move the directory creation piece out of it; or disable it here somehow. It will create the default directories with no arguments - so you will still get the debug directories even if you are building release, etc.